bin/deploy-dev.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin bash
```

Comment:
how about `postdeploy-dev.sh` name?
I agree this script does not do any deployment